internal/manager/controllerdef.go

```diff
@@ -255,7 +255,7 @@ func setupControllers(
 		// Gateway API Controllers - Beta APIs
 		// ---------------------------------------------------------------------------
 		{
-			Enabled: featureGates[featuregates.GatewayFeature],
+			Enabled: true,
```

Comment:
Are we supposed to have it enabled by default without an opt-out option? For others controllers KIC offers granular control e.g. `IngressNetV1`, `KongConsumer`, etc. 🤔

Reply:
yeah I guess we do need to flags to allow disabling those controllers just like we do with all the others

Reply:
This is now based on the cli flags that were introduced in https://github.com/Kong/kubernetes-ingress-controller/pull/4971